client/my-sites/stats/site.jsx

```diff
@@ -193,7 +219,7 @@ class StatsSite extends Component {
 					siteId={ siteId }
 					slug={ slug }
 				/>
-
+				{ privateSiteBanner }
```

Comment:
Tiny nit: why not add whitespace before and after the new content?

Separately, why not do the call inline?
```suggestion

				{ isSitePrivate ? this.renderPrivateSiteBanner() : null }

```